getting-started/setup-building.rst

```diff
@@ -483,31 +483,47 @@ Regenerate ``configure``
 
 If a change is made to Python which relies on some POSIX system-specific
 functionality (such as using a new system call), it is necessary to update the
-``configure`` script to test for availability of the functionality.
+:file:`configure` script to test for availability of the functionality.
+Python's :file:`configure` script is generated from :file:`configure.ac`
+using GNU Autoconf.
 
-Python's ``configure`` script is generated from ``configure.ac`` using Autoconf.
-Instead of editing ``configure``, edit ``configure.ac`` and then run
-``autoreconf`` to regenerate ``configure`` and a number of other files (such as
-``pyconfig.h``).
+After editing :file:`configure.ac`, run ``make regen-configure`` to generate
+:file:`configure`, :file:`pyconfig.h.in`, and :file:`aclocal.m4`.
+When submitting a patch with changes made to :file:`configure.ac`,
+you must also those generated files.
 
-When submitting a patch with changes made to ``configure.ac``, you should also
-include the generated files.
+The recommended and by far the easiest way to regenerate :file:`configure` is::
 
-Note that running ``autoreconf`` is not the same as running ``autoconf``. For
-example, ``autoconf`` by itself will not regenerate ``pyconfig.h.in``.
-``autoreconf`` runs ``autoconf`` and a number of other tools repeatedly as is
-appropriate.
+   $ make regen-configure
 
-Python's ``configure.ac`` script typically requires a specific version of
-Autoconf.  At the moment, this reads: ``AC_PREREQ(2.69)``. It also requires
-to have the ``autoconf-archive`` and ``pkg-config`` utilities installed in
-the system and the ``pkg.m4`` macro file located in the appropriate ``alocal``
-location. You can easily check if this is correctly configured by running::
+This Make target will run a Podman or Docker container (depending on your
+environment); you only need to commit the changes.
+Behind the scenes this Make target runs one of the following container images::
```

Comment:
Is it useful to talk about what the make target does? Easy enough to check for oneself if curious.

Reply:
It is useful if you're hacking in a cleaned repo (there's no `Makefile` for you to run the `regen-configure` target yet). I could mention that.

Reply:
Suggestion here: https://github.com/python/devguide/pull/1109/files#r1223569962

Reply:
PTAL